src/renderer/components/app.tsx

```diff
@@ -69,6 +72,32 @@ export class App extends React.Component {
     whatInput.ask(); // Start to monitor user input device
   }
 
+  async componentDidMount() {
+    console.log("hello");
+    const cluster = getHostedCluster();
+    await eventStore.loadAll();
+    await nodesStore.loadAll();
+    await podsStore.loadAll();
```

Comment:
If these are not dependent of each other then should be done concurrently.

```suggestion
    await Promise.all([
      eventStore.loadAll(),
      nodesStore.loadAll(),
      podsStore.loadAll(),
    ]);
```

Reply:
They are not, I was mainly worried about hitting kubernetes api too hard (if these collections are huge). But maybe it's better to do these concurrently.

Reply:
Actually, I see a problem here.. we need to check if user has access to these resources before trying to load them.

Reply:
Fixed.